src/components/AddCustomerButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { Button } from '@patternfly/react-core';
+import { useQueryClient } from 'react-query';
+
+interface Props {
+  setIsModalOpen: (isOpen: boolean) => void;
+}
+
+const AddCustomerButton: React.FC<Props> = ({ setIsModalOpen }) => {
+  const queryClient = useQueryClient();
+
+  const handleButtonClick = () => {
+    setIsModalOpen(true);
+  };
+
+  return (
+    <Button onClick={handleButtonClick} variant='secondary'>
+      Add New Customer
+    </Button>
+  );
+};
+
+export default AddCustomerButton;
```

Comment:
To recap what we discussed in slack.  the button you need is already present. the goal is to be able to add a customer and have it pop up on the screen relatively immediately

Reply:
i think this file can be deleted.

Reply:
This file can still be removed